bitnami/airflow/templates/config/configmap.yaml

```
@@ -14,7 +14,7 @@ metadata:
 data:
 {{- if .Values.config }}
   airflow.cfg: |-
-    {{ .Values.config | indent 4 }}
+{{ .Values.config | indent 4 }}
```

```suggestion
    {{ .Values.config | nindent 4 }}
```
I mean to use this. `nindent` allows to put spaces before the line and it will indent the number indicated no mater where the line starts.

---

i see... updated as per request. thanks!